apis/namespaced/v1alpha1/doc.go

```diff
@@ -4,6 +4,6 @@
 
 // Package v1alpha1 contains the core resources of the aws provider.
 // +kubebuilder:object:generate=true
-// +groupName=m.aws.upbound.io
+// +groupName=aws.m.upbound.io
```

Comment:
Could we switch everything to use `aws.m.upbound.io`?

I think that actually makes the most sense - since `m.upbound.io is the least specific part of the domain.

Reply:
sure, we can do that! that actually makes more sense for the root "config" group, as the `xpkg batch` assumes all of the CRDs for that group start with `aws.*`. Let's see how code gen like this update... 🙏

Reply:
This is now complete @negz, all the namespaced CRDs consistently use `*.aws.m.upbound.io` and I manually built/tested with the s3 provider 👍